syntax = "proto3";

enum BookCategory {
    MYSTERY = 0;
    SCIENCE_FICTION = 1;
    SELF_HELP = 2;
}

message InventoryRequest {
    int32 user_id = 1;
}

message Book {
    int32 id = 1;
    string title = 2;
}

message InventoryResponse {
    repeated Book books = 1;
}

service InventoryBooks {
    rpc Inventory (InventoryRequest) returns (InventoryResponse);
}